src/frameworks/django/index.ts

```diff
@@ -0,0 +1,86 @@
+import { copy, mkdirp, pathExists } from "fs-extra";
+import { mkdir, readFile, readdir, writeFile } from "fs/promises";
+import { join } from "path";
+import { BuildResult, FrameworkType, SupportLevel } from "../interfaces";
+import { dirExistsSync } from "../../fsutils";
+import { findPythonCLI, hasPipDependency, spawnPython } from "../utils";
+import { spawnSync } from "child_process";
```

Comment:
spawn should come from `cross-spawn` to prevent issues with Windows
```suggestion
import { sync as spawnSync } from "cross-spawn";
```